README.md

```diff
@@ -34,7 +34,7 @@ import com.outr.lucene4s._
 
 ```scala
 val directory = Paths.get("index")
-val lucene = new Lucene(directory = Option(directory))
+val lucene = new DirectLucene(Nil, directory = Option(directory))
```

Comment:
Why the switch from the generic to explicit use of `DirectLucene`?

Reply:
`Lucene` is a trait, you can't instantiate it. Happy to put something else there but I don't know what is best. It's always frustrating when you go to a new project and the examples don't work.